orchagent/response_publisher.cpp

```diff
@@ -64,26 +64,52 @@ void RecordResponse(const std::string &response_channel, const std::string &key,
 
 } // namespace
 
-ResponsePublisher::ResponsePublisher(bool buffered)
```

Comment:
Can you please provide the details of response_publisher in the description? What was the behavior before and whats the new change? Is it breaking any previous implementation?

Reply:
Updated the PR description.